File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/8 - 6 and 7 April 2019/03. Oscars week in cinema.py
film_name = input()
type_saloon = input()
count_tickets = int(input())

price = 0

if film_name == "A Star Is Born":
    if type_saloon == "normal":
        price = 7.50
    elif type_saloon == "luxury":
        price = 10.50
    elif type_saloon == "ultra luxury":
        price = 13.50
elif film_name == "Bohemian Rhapsody":
    if type_saloon == "normal":
        price = 7.35
    elif type_saloon == "luxury":
        price = 9.45
    elif type_saloon == "ultra luxury":
        price = 12.75
elif film_name == "Green Book":
    if type_saloon == "normal":
        price = 8.15
    elif type_saloon == "luxury":
        price = 10.25
    elif type_saloon == "ultra luxury":
        price = 13.25
elif film_name == "The Favourite":
    if type_saloon == "normal":
        price = 8.75
    elif type_saloon == "luxury":
        price = 11.55
    elif type_saloon == "ultra luxury":
        price = 13.95

price = price * count_tickets

print(f"{film_name} -> {price:.2f} lv.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/05. Online Exam - 6 and 7 April 2019/03. Oscars week in cinema.js
function oscarsWeekInCinema(params) {
    let name = params.shift();
    let room = params.shift();
    let tickets = Number(params.shift());

    let prices = {
        'A Star Is Born': {'normal': 7.50, 'luxury': 10.50, 'ultra luxury': 13.50},
        'Bohemian Rhapsody': {'normal': 7.35, 'luxury': 9.45, 'ultra luxury': 12.75},
        'Green Book': {'normal': 8.15, 'luxury': 10.25, 'ultra luxury': 13.25},
        'The Favourite': {'normal': 8.75, 'luxury': 11.55, 'ultra luxury': 13.95}
    };

    let price = prices[name][room] * tickets;

    console.log(`${name} -> ${price.toFixed(2)} lv.`);
}


oscarsWeekInCinema(['A Star Is Born', 'luxury', '42']);  // Expected Output:
// A Star Is Born -> 441.00 lv.

oscarsWeekInCinema(['Green Book', 'normal', '63']);  // Expected Output:
// Green Book -> 513.45 lv.

oscarsWeekInCinema(['The Favourite', 'ultra luxury', '34']);  // Expected Output:
// The Favourite -> 474.30 lv.